webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue

```diff
@@ -85,18 +90,8 @@ export default {
     getProviders() {
       // services object contains urls for requests
       try {
-        const thisvue = this;
         webConferencing.getProvidersConfig().then((providersConfig) => {
-          for (const providerConfig of providersConfig) {
-            webConferencing.getProvider(providerConfig.type).then((provider) => {
-              providerConfig.provider = provider;
-              // set setting visibility for the provider
-              providerConfig.hasSettings = provider.showSettings && provider.hasOwnProperty('showSettings');
-              thisvue.providersConfig.push(providerConfig);
-            }).fail(function(err) {
-              thisvue.log.warn(`Provider ${providerConfig.type} is not available`, err);
-            });
-          }
+          this.providersConfig = providersConfig;
```

Comment:
Why did you remove this ?

Reply:
still a quesiton here

Reply:
this is removed because it was used to display connector configuration (only webrtc have it).
As webrtc will be remove, we can remove this part.